src/components/List/List.js

```diff
@@ -1,7 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { firestore } from '../../lib/firebase.js';
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+import GroceryItem from '../GroceryItem/GroceryItem';
 
 const List = () => {
-  return <div>Current List</div>;
+  const token = 'token';
```

Comment:
```suggestion
  const token = props.token;
```

Reply:
Taryn and I pass the token down as props to our List component, so we'll be able to access it here with props.token.